lib/stats/stats-query.c

```diff
@@ -182,6 +192,95 @@ _query_counter_hash(gchar *key_str)
   return counters;
 }
 
+static GList *
+_get_input_counters_of_view(ViewRecord *view)
+{
+  GList *counters_of_view = NULL;
+  for (GList *q = view->queries; q; q = q->next)
+    {
+      GList *selected_counters = NULL;
+      gchar *query = q->data;
+
+      selected_counters = _query_counter_hash(query);
+      if (selected_counters == NULL)
+        continue;
+
+      counters_of_view = g_list_concat(counters_of_view, selected_counters);
+    }
+  return counters_of_view;
+}
+
+static GList *
+_get_aggregated_counters_from_views(GList *views)
```

Comment:
It can be const because it is iterated over inside the function, but the compiler gives a warning.